ptrace/syscall/prototypes.py

```diff
@@ -1250,11 +1248,11 @@
         ("const char *", "uargs"),
     )),
     "select": ("long", (
-        ("int", "n"),
-        ("fd_set *", "inp"),
-        ("fd_set *", "outp"),
-        ("fd_set *", "exp"),
-        ("struct timeval *", "tvp"),
+        ("int", "nfds"),
+        ("fd_set *", "readfd"),
```

Comment:
Would you mind to add an S to readfd, writefs and exceptfd parameter names?

Reply:
I justed pushed a commit doing so !